src/inference/adapt/stepsize.jl

```diff
@@ -87,15 +87,16 @@ function adapt_stepsize!(da::DualAveraging, stats::Real)
     η_x = m^(-κ)
     x_bar = (1.0 - η_x) * x_bar + η_x * x
 
-    ϵ = exp(x)
+    ϵ = exp(x_bar)
```

Comment:
Can you explain this change a bit more? I re-read the NUTS paper (Algorithm 6 of https://arxiv.org/pdf/1111.4246.pdf) and the Stan code (https://github.com/stan-dev/stan/blob/develop/src/stan/mcmc/stepsize_adaptation.hpp#L83) - both of them seem to do what was done in Turing before (i.e. `ϵ = exp(x)`). What's exactly going on here?

Reply:
sorry; the old version is correct -  I was confused by `ϵ_NAdapt` and `ϵ`. The actual bug was fixed by 

https://github.com/TuringLang/Turing.jl/blob/c27e88f9dc1d00089021491fe62a9d1c85072137/src/inference/adapt/stepsize.jl#L99

ESS for `m12_2` gets notificablely better than current after reverting this to `ϵ = exp(x)`.